internal/provider/provider.go

```diff
@@ -164,6 +165,8 @@ func (p *ProviderFramework) DataSources(ctx context.Context) []func() datasource
 		vaultsecrets.NewVaultSecretsSecretDataSource,
 		// IAM
 		iam.NewServicePrincipalDataSource,
+		// // Waypoint
+		// waypoint.NewApplicationTemplateDataSource,
```

Comment:
I forgot to reintroduce this, I'll update the PR later after the test issue is resolved